mysql/datadog_checks/mysql/mysql.py

```diff
@@ -985,6 +985,13 @@ def _get_stats_from_innodb_status(self, db):
                 "Privilege error or engine unavailable accessing the INNODB status tables (must grant PROCESS): %s", e,
             )
             return {}
+        except (UnicodeDecodeError, UnicodeEncodeError) as e:
+            self.log.warning(
+                "Unicode error while getting INNODB status "
+                "(if this warning is infrequent, metric collection won't be impacted): %s",
```

Comment:
(Nit) Actually, WDYTA dropping the double negation? 😅 

```suggestion
                "(typically harmless, but if this warning is frequent metric collection could be impacted): %s",
```